packages/communication-react/CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ Fri, 09 Jul 2021 20:41:33 GMT
 
 ### Changes
 
+* Updated calling sdk dependency to 1.2.0-beta.1
+
+* Updated Calling UI Components with UI updates
+
+* Updates to selectors and adapters. Learn more about these updates in aka.ms/acsstorybook
+
+* Bundle optimizations when only using chat or only using calling components
+
 - Bump react-components to v1.0.0-beta.2 ([PR #496](https://github.com/azure/communication-ui-library/pull/496) by jinan@microsoft.com)
 
```

Comment:
Couple can we add:
* Updated react peer dependency to >=16.8.0 <18.0.0
* Calling composite improvements including floating local video tile, permission banner when call is being recorded, changing local video device and microphone, and screensharing view improvements
* Chat composite improvements including notifying a participant when they leave a thread and styling improvements.

Reply:
Maybe also mention "lots of documentation improvements at aka.ms/acsstorybook"